docs/reference/tags/item_selections.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: default
+title: Item Selections
+parent: Tags
+has_children: false
+---
+
+The Item Selections Tag returns an object which can be used as a wrapper to display modifiers for a given item.
+
+
+##### input
+{% raw %}
+```liquid
+    {% form 'item_selections', item %}
+    {% endform %}
+```
+{% endraw %}
+
+##### output
+{% raw %}
+```html
+<FormTag::ItemSelectionsForm:0x000000010cf77300>
```

Comment:
This isn't what's rendered on the page is it? Should we show the actual form tags that will be rendered?

Reply:
Agreed, I need to find an example of the form tag being used. 
@paulina-a, do you have any code snippets you can share?

Reply:
Here is an example of where we use the form: https://github.com/easolhq/alchemist-theme/blob/master/templates/selections/index.html

Reply:
@paulina-a any final notes for this PR?

Reply:
I don't fully understand this one to be honest! Lets merge as is if tom is happy with it and then we can always come back and add more detail at a later date if needed. Thanks @JHarrison89 !